client/src/test/java/com/kritsit/casetracker/shared/domain/model/IncidentTest.java

```diff
@@ -81,10 +81,8 @@ public void testEquals_Class() {
     }
 
     public void testEquals() {
-        LocalDate date = LocalDate.now();
```

Comment:
This is the merge conflict. I noticed this when I was working today and pushed the change. You are correct that it was unintended.